test/franklin/articles_test.exs

```diff
@@ -29,15 +29,10 @@ defmodule Franklin.ArticlesTest do
       assert {:ok, ^uuid} = Articles.create_article(max_attrs)
 
       assert_receive {:article_created, %{id: ^uuid}}
-      assert_receive {:article_body_updated, %{id: ^uuid}}
-      assert_receive {:article_slug_updated, %{id: ^uuid}}
-      assert_receive {:article_published_at_updated, %{id: ^uuid}}
-      assert_receive {:article_title_updated, %{id: ^uuid}}
```

Comment:
Since the create event not has all the attributes of the article, we no longer publish all these pubsub named notifications.